pkg/replication/replicator.go

```diff
@@ -14,6 +16,10 @@ import (
 
 const DefaultConcurrentReality = 3
 
+// Normal replications will have no timeout, but daemon sets will
+// because it is unlikely that all hosts are healthy at all times
```

Comment:
@abishopric asked to specify a time limit because it was possible that some hosts will never be healthy and to log the hosts which did not meet the timeout.